extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputFormat.java

```diff
@@ -59,6 +69,35 @@ public boolean isSplittable()
   @Override
   public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity source, File temporaryDirectory)
   {
+    try {
+      KafkaRecordEntity kafkaInputEntity = (KafkaRecordEntity) source;
+      Header versionHeader = kafkaInputEntity.getRecord().headers().lastHeader(VERSION_HEADER_KEY);
+      int version = 0;
```

Comment:
no need to declare the version here, since we don't use it outside of the scope of the following if statement.

Reply:
right, fixed it